csrankings-7.csv

```diff
@@ -2041,7 +2041,7 @@ Sebastian Alexander Mödersheim,DTU,https://www.compute.dtu.dk/english,NOSCHOLAR
 Sebastian Altmeyer,University of Augsburg,https://www.uni-augsburg.de/de/fakultaet/fai/informatik/prof/embsys/team/prof-dr-sebastian-altmeyer,O5JtLZsAAAAJ
 Sebastian Angel,University of Pennsylvania,https://cis.upenn.edu/~sga001,pNaNj8EAAAAJ
 Sebastian Baltes,University of Adelaide,https://empirical-software.engineering,xO09KrYAAAAJ
-Sebastian Boring,University of Copenhagen,http://www.diku.dk/english/staff/?pure=en/persons/431157,Y_QMLKAAAAAJ
+Sebastian Boring,Aalborg University,http://sebastianboring.com/,Y_QMLKAAAAAJ

```

Comment:
Not in a CS department, not listed here: https://vbn.aau.dk/da/organisations/institut-for-datalogi/persons/

Reply:
Listed here: https://vbn.aau.dk/da/persons/141070

Reply:
That page does not show affiliation with a CS department.

Reply:
I suppose it's adjacent but not quite CS, true. I removed that entry from csrankings-7.csv